HarvardAgileRoboticsLab/drake
# Kuka Director-Dircol

Finishes the integration between director and dircol. We are using the Matlab implementation of dircol, we will move to a C++ implementation pending some updates to drake.

Test instructions:
- Start procman with the dircol pmd file:
`./build/install/bin/bot-procman-sheriff -l drake/examples/kuka_iiwa_arm/kuka_sim_with_dircol.pmd`
- Start a matlab process and add drake to the path (the usual `addpath_drake`)
- Run the `runKukaPlanner` script
- From procman run the `3. kuka-dircol-planner` processes and the director app should open.
- You should be able to interact with the arm by dragging it around and plan a dynamic trajectory by clicking the plan button. The planning might take a couple of seconds, so be patient.
